doc/netplan-generate.md

```diff
@@ -64,7 +64,7 @@ There are 3 locations that **`netplan generate`** considers:
 
 If there are multiple files with exactly the same name, then only one
 will be read. A file in `/run/netplan` will shadow - completely replace
```

Comment:
```suggestion
will be read. A file in `/run/netplan` will shadow (completely replace)
```

Reply:
That makes sense and seems easy to understand! I tried to commit your suggestion. I don't really know how to use github and am a complete noob. But that change seems good. I'm very happy if this was at all helpful.

Reply:
Oh yes, it was. Thanks a lot for your contribution to Netplan. And the commit went through and is now merged into Netplan's main branch!